tests/containers/skopeo.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: skopeo
+# Summary: Test basic skopeo commands.
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';    # used in select_serial_terminal
+use utils 'zypper_call';    # used in zypper_call
+use version_utils 'package_version_cmp';
+use version_utils 'is_transactional';
+use transactional;
+use containers::common qw(install_packages);
+
+sub run {
+    my ($self, $args) = @_;
+
+    ######### Set required variables
+
+    # Required packages
+    my @packages = qw(skopeo jq);
+
+    select_serial_terminal();    # Select most suitable text console
+
+    # Set a variable for my remote image
+    my $remote_image = 'registry.suse.com/bci/bci-busybox:latest';
+    # Set a variable for my working directory
+    my $workdir = '/tmp/test';
+
+    ######### Ensure pre-requisites
+
+    # Install required packages
+
+    # install_packages accounts for SLE-Micro environment with transactional-update
+    install_packages(@packages);
+
+    # Skopeo version emits a reasonable output
+    # Verify that skopeo version is greater or equal to 1.12.0.
+    die('wrong skopeo version') unless (package_version_cmp('1.12.0', script_output('skopeo --version | cut -d" " -f3')) <= 0);
+    # skopeo --version | cut -d" " -f3 --> separates "skopeo version 1.12.0" output into and array and takes third value "1.12.0"
+
+    # Create test directory
+    script_run("mkdir $workdir");
+
+    ######### Inspect a docker repository
+
+    # assert_script_run --> runs the command and die unless it returns zero, indicating successful completion of $cmd
+    assert_script_run("skopeo inspect docker://$remote_image",
+        fail_message => 'Failed to inspect remote image.');
+
+
+    ######### Pull the image into a directory
+
```

Comment:
There are so many unnecessary spaces. 
I also would put those titles into `record_infos` to help debugging if it fails, for example
`record_info('Test 1', 'Install required packages')`
`record_info('Test 2', 'Inspect a docker repository')`
etc..

Reply:
Agree.

Reply:
Thanks! Added record_info all around, and cleaned up any unnecessary spaces.